help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,245 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Identity Resolution Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity Resolution destionation connection using the Adobe Experience Platform UI.
+hide: true
+hidefromtoc: true
+---
+# Merkury Enterprise Connections Destination
+
+Clients of Adobe Experience Platform use the Merkury Enterprise
+Connections Destination to securely deliver media audiences to Merkury.
+Merkury provides marketers easy matching and delivery of person-based
+audiences to Merkury's 80+ premium addressable TV/CTV, publisher, and
+ad-tech connections. Merkury expands on Adobe's native destination
+endpoints to provide marketers more options to reach their customers and
+prospects with cookie-less addressable media. Merkury is powered by a
+comprehensive US adult consumer identity graph of 268+ million people.
+The Merkury identity graph provides an average of 15% improvement in
+match rate when activating media audiences using Merkury.
+
+![A diagram of a customer data platform Description automatically
+generated](./merkury-connections/media/image1.png){width="6.5in"
+height="3.946527777777778in"}
+
+Follow these steps to create an Merkury Connections destination
+connection and dataflow using the Adobe Experience Platform user
+interface.![A screenshot of a computer Description automatically
```

Comment:
the alt-text is not useful, it should describe what's happening in the screenshot.